sif_parser/sif_open.py

```diff
@@ -150,3 +151,119 @@ def xr_open(sif_file, ignore_corrupt=False, lazy=None):
     return xr.DataArray(data, dims=['Time', 'height', 'width'],
                         coords=coords, attrs=new_info)
 
+def np_spool_open(spool_dir, ignore_missing=False, lazy=None):
+    """
+    Read from a directory the binary files and metadata generates via spooling and return a np.array. 
```

Comment:
Maybe 
"Read the binary files and meta data from the directory generated via the spooling acquisition. Returns a np.array and a dictionary of the meta data"?